lib/tenbotter/team.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Team
```

Comment:
from my other comment, here's my suggestion:

```ruby
class Team
  class << self
    def draft(players)
      team_one = Team.new
      team_two = Team.new
      team_one.captain, team_two.captain = players.shift(2)
      team_one << team_one_captain
      team_two << team_two_captain
      current_team_picking= %i[team_one team_two].shuffle

      while players_by_ranking.count.positive?
        shift_amount = if players_by_ranking.count == 8 || players_by_ranking.count == 1
                       1
                     else
                       2
                     end

        players_to_add = players_by_ranking.shift(shift_amount)

        if current_team_picking == :team_one
          team_one.players << *players_to_add
          current_team_picking = :team_two
        elsif current_team_picking == :team_two
          team_two.players << *players_to_add
          current_team_picking = :team_one
        end
      end

      [team_one, team_two]
    end
  end

  attr_accessor :players
  attr_reader :captain

  def initialize(players = [])
    @players = players
  end

  def captain=(new_captain)
    captain = new_captain
    unless players.include?(new_captain)
      players << new_captain
    end
  end
end
```

Reply:
Don't you think drafting is moreso associated with a Game than with a Team?

Reply:
I'm on the edge, I kinda think it makes sense to have it on the Team, but I don't really have any strong inclination. So down to go with what you have, but just wanted to get your thoughts